crypto/rsa/rsa_lib.c

```diff
@@ -409,6 +415,9 @@ int RSA_set0_multi_prime_params(RSA *r, BIGNUM *primes[], BIGNUM *exps[],
             pinfo->r = primes[i];
```

Comment:
could you please also change the BN_free above this line to BN_clear_free (3 times)?